### DISCUSSION TITLE
2024.06.07 Town Hall

### DISCUSSION THREAD

**Discussion by meg-gutshall** [Virtual-Coffee/virtualcoffee.io#1198] [Town Halls]
Hello Virtual Coffee members!

Below are the minutes from the VC Town Hall held earlier this month. Please add any thoughts, comments, questions, etc. in a thread on this post and we will continue discussion here. If you wish to provide feedback anonymously, you can use the [Virtual Coffee Town Hall form](https://forms.gle/f7ngSFTqydLbJ4hA8) at any time and your submission will be addressed in the subsequent meeting.

Our next Town Hall is currently scheduled for Friday, September 6th at 1:00 PM ET.

---

# Virtual Coffee Town Hall – June 6, 2024

## Introduction

- We’ve been doing Virtual Coffee since April 2020 (maybe March 2020)
- The current maintainers will meet following this session to go over input from today and review our goals

### VC Maintainer History

- The original maintainers included Kieran Black, Bekah Hawrot-Weigel, Dan Ott, and Kirk Shillingford
- Kieran Black stepped down as maintainer
- Julia Seidman became a maintainer (March 2023)
- Meg Gutshall became a maintainer and we restructured maintainers to have roles (March 2024)
  - Community Health Maintainer → Kirk
  - Events Maintainer → Meg
  - Communications Maintainer → Bekah
  - Early Career Maintainer → Julia
  - Technology Support Maintainer → Dan
- Julia stepped down as maintainer (May 2024)
  - We’re still figuring out how to distribute the Early Career Maintainer duties

### Open sourcing the [VC-Contributors repo](https://github.com/Virtual-Coffee/VC-Contributors)

- It was initially (and still is) private to keep it visible only to VC members
- We want to be more transparent with documentation about what we do and how we operate
- Why? We want to be able to give an example/framework for other groups to follow and start their own communities – empowerment!!
- Bekah & Ayu are leading this initiative with lots of help from Dominic & Kirk

### Communication

- Slack, [GitHub discussions](https://github.com/orgs/Virtual-Coffee/discussions), [website](https://virtualcoffee.io/) ([virtualcoffee.io repo](https://github.com/Virtual-Coffee/virtualcoffee.io)), [VC-Contributors repo](https://github.com/Virtual-Coffee/VC-Contributors)
- We need to figure out a single source of truth

### Coffee Table Events

- What do people want to see??
  - What are our members interested in seeing, especially when it comes to the Tech Interview Study Group event?
- Alternate times to accommodate different time zones
  - We need to figure out what our membership spread is across time zones

### Tuesday/Thursday VC Meetings

- Add another VC meeting to accommodate a third time zone
- Current room leaders/note takers should reach out to Bekah if they’re interested in volunteering for a third time zone that’s more accommodating to members in Europe and Asia

### Sponsorships

- They’re hard to get without advanced signups
- We need to start tracking data and metrics – that’s what sponsors want to see

### Discord vs. Slack

- Discoverability is very difficult
  - A suggestion: Shut down all Slack channels except one with instructions on how to move to Discord
- Many people prefer the disappearing messages that Slack facilitates
- GitHub discussions is for permanent conversations

### Website & CMS

- The website needs an overhaul so people can contribute
- The content on the website needs to be more discoverable
  - We need a redesign

### Streaming

- We have Streamyard setup to stream to Twitter, Twitch, and YouTube
- Contact Meg if you want to set something up
- Nick volunteered to co-stream with anyone who’s interested

### Hackathon

- Doesn’t have to be an explicit hackathon
- We can partner with Ruby for Good if people are interested in that

### A bigger monthly challenge every three months

- Move NaNoWriMo to May? ([WeblogPoMo](https://weblog.anniegreens.lol/weblog-posting-month-2024))

### Partner up with other communities

- Self-Taught Software Developers (Leader: Fifi Degarr)
- Learn. Build. Teach. (Leader: James Quick)
- JS Craft Camp (Org Team: Jörn Bernhardt)

## Action Items

- [ ] Nick – Will talk with Dan about helping with the website overhaul
  - Joe, Jono – Will help as well
- [ ] Open issues on the website repo! We need to start prepping for Hacktoberfest!

## Next Meeting's Agenda

- Summarize community changes since last meeting
- List any outstanding open items
- Open the floor for community questions, feedback, and discussion
- Address any items submitted through the [Virtual Coffee Town Hall form](https://forms.gle/f7ngSFTqydLbJ4hA8) that weren’t addressed during the course of open discussion

**Comment by cmcrawford2** [comment]
I'm not sure what the problem is with Slack. Maybe the maintainers know more than I do. I have seven slack workspaces that I look at every day. I don't have discord on my computer. If we switch to discord, that means I have to download it, and then look at VC separately from everything else that's work-related. That doesn't really work for me.

**Reply by BekahHW** [reply]
Cris, I also prefer slack. I don't think we've ever _not_ had this brought up at a townhall/maintainer Q&A. It's a consistent conversation. Discord does have better tools for community moderation and for events. Personally, I do not feel we've reached a point where we need to make this switch, but I'm always open to hearing others' viewpoints.

**Reply by SuperRoach** [reply]
Can agree with the slack vs x arguments are difficult. Our local hackerspace ended up moving from Slack to Mattermost. Mattermost is something that can be self hosted for free, but that brings the difficulty that you need to have someone help keep that up to date long term.
